pkg/cli/cvd_test.go

```diff
@@ -72,41 +72,38 @@ func TestAdditionalInstancesNum(t *testing.T) {
 	}
 }
 
-func TestGetHostOutX8664(t *testing.T) {
-	buildTop := "foo"
+func TestGetHostOutRelativePathX8664(t *testing.T) {
 	targetArch := "x86_64"
 
-	got, err := GetHostOut(buildTop, targetArch)
+	got, err := GetHostOutRelativePath(targetArch)
 
 	if err != nil {
 		t.Fatal(err)
 	}
-	expected := "foo/out/host/linux-x86"
+	expected := "out/host/linux-x86"
 	if diff := cmp.Diff(expected, got); diff != "" {
 		t.Errorf("mismatch (-want +got):\n%s", diff)
 	}
 }
 
-func TestGetHostOutArm64(t *testing.T) {
-	buildTop := "foo"
+func TestGetHostOutRelativePathArm64(t *testing.T) {
 	targetArch := "arm64"
 
-	got, err := GetHostOut(buildTop, targetArch)
+	got, err := GetHostOutRelativePath(targetArch)
 
 	if err != nil {
 		t.Fatal(err)
 	}
-	expected := "foo/out/host/linux_musl-arm64"
+	expected := "out/host/linux_musl-arm64"
 	if diff := cmp.Diff(expected, got); diff != "" {
 		t.Errorf("mismatch (-want +got):\n%s", diff)
 	}
 }
```

Comment:
Golang encourages the use of table driven tests https://go.dev/wiki/TableDrivenTests#introduction, which is a good fit in this case.

However, I think verifying all values within the internal lookup table is the same as writing the table itself. Due the simplicity of the function under test, I think there's no need for unit test. However if adding unit tests I think we should test two behaviors only, when the target arch is supported and when it's not rather than verifying all the the arch values.

Reply:
Done, but just left 2 unit test; one for known arch and the other for unknown arch.